GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -225,16 +225,42 @@ protected override void Dispose(bool disposing)
         {
             if (disposing)
             {
+                //// MenuCommands not disposable
+                //// _superprojectCurrentCheckout not disposable
+                //// _selectedObjectIds not disposable
+                //// _baseCommitToCompare not disposable
                 _revisionSubscription?.Dispose();
                 _revisionReader?.Dispose();
+                //// _artificialStatus not disposable
+                //// _ambiguousRefs not disposable
+                //// _refFilterOptions not disposable
+                _lastVisibleResizableColumn?.Dispose();
```

Comment:
❕ `_lastVisibleResizableColumn` is not owned by this control and must not be disposed directly here.